README.md

```diff
@@ -1,5 +1,5 @@
 ![elpho logo][logo]
-Extension Library for PHp Object-orientation
```

Comment:
The name had this presentation to keep only the letters composing the name "ELPHO" in uppercase.
If you find another form to emphasize it you're welcome.